### PR TITLE
Target doc updates

### PR DESCRIPTION
**Pull request by swarna04**

### CODE REVIEW

File: using-mobile-extensions/adobe-target/README.md [hunk 1]
```diff
@@ -357,136 +252,116 @@ purchasedProductIds.add("125");
...
+`TargetParameters` including `mboxParameters`, `profileParameters`, `orderParameters` and `productParameters` can be passed in Target APIs and also when creating `TargetPrefetch` or `TargetRequest` objects. The `TargetParameters` passed in public APIs are considered global parameters and are merged with the corresponding parameters in the individual `TargetRequest` or `TargetPrefetch` objects.
 
-You can also set an app deep link that can be triggered when selections are made in the preview mode by using the following methods:
+While merging, the new keys in the mbox parameters or the profile parameters are appended to the final dictionary while the keys with the same name are overwritten in each `TargetRequest` or `TargetPrefetch` object by those from global parameters. In the case or `TargetOrder` or `TargetProduct` objects, the object passed in global parameters shall replace the corresponding object in each of `TargetRequest` or `TargetPrefetch` objects.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target/README.md]: Please reword as follows:
When merging, the new keys in the mbox parameters or the profile parameters are appended to the final dictionary, and  the keys with the same name are overwritten in each `TargetRequest` or `TargetPrefetch` object by the keys from the global parameters. For `TargetOrder` or `TargetProduct` objects, the object that is passed to the global parameters replaces the corresponding object in the `TargetRequest` or `TargetPrefetch` objects."

File: using-mobile-extensions/adobe-target/README.md [hunk 2]
```diff
@@ -357,136 +252,116 @@ purchasedProductIds.add("125");
...
 
-{% tabs %}
-{% tab title="Android" %}
-#### **Syntax**
+## Target Sessions
 
-```java
-public static void setPreviewRestartDeepLink(final Uri deepLink);
-```
+With Target release v2.1.4 iOS and  v1.1.3 Android, Target extension now supports persistent sessions. When a Target request is received, a new session ID is generated if one doesn't already exist and is sent in the request. This session ID along with the Edge Host returned from the Target are stored in the persistent storage for the configured `target.sessionTimeout`. If not configured, the default value for the session timeout is 30 minutes. These variables are reset and removed from persistent storage if no Target request is received during the configured `target.sessionTimeout` or if  [resetExperience API](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#reset-user-experience) is called.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target/README.md]: Please reword as follows:
"The Target extension (version 2.1.4 for iOS) and  (version 1.1.3 for Android) now supports persistent sessions. When a Target request is received, if a session ID does not exist, a new ID is generated and is sent in the request. This ID, with the Edge Host that is returned from the Target, is kept in persistent storage for the configured `target.sessionTimeout`. If the timeout value is not configured, the default value is 30 minutes. If no Target request is received during the configured `target.sessionTimeout` or if  the [resetExperience](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#reset-user-experience) API is called, these variables are reset and removed from persistent storage ."

(please don't include "API" in the link...only the actual API name should be in the link.)

File: using-mobile-extensions/adobe-target/README.md [hunk 3]
```diff
@@ -357,136 +252,116 @@ purchasedProductIds.add("125");
...
-
-```java
-Target.setPreviewRestartDeepLink("myApp://HomePage");
-```
-{% endtab %}
+## Visual preview
 
-{% tab title="iOS" %}
-#### Syntax
+Visual preview mode allows you to easily perform end-to-end QA for Target activities by enrolling and previewing these activities on your device. This mode does not require a specialized testing set up. To get started, set up a URL scheme and generate the preview links. For more information, see [Target mobile preview](https://docs.adobe.com/content/help/en/target/using/implement-target/mobile-apps/target-mobile-preview.html).
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target/README.md]: - please reword as follows:
"The visual preview mode allows you to easily perform end-to-end QA activities by enrolling and previewing these activities on your device. This mode does not require a specialized testing set up. To get started, set up a URL scheme and generate the preview links. For more information, see [Target mobile preview](https://docs.adobe.com/content/help/en/target/using/implement-target/mobile-apps/target-mobile-preview.html)."

(What is a URL scheme? Is this a commonly understood term?)

File: using-mobile-extensions/adobe-target/README.md [hunk 4]
```diff
@@ -357,136 +252,116 @@ purchasedProductIds.add("125");
...
 
-```text
-+ (void) setPreviewRestartDeepLink: (nonnull NSURL*) deepLink;
-```
+You can also set an application deep link that can be triggered when selections are made in the preview mode by using the [setPreviewRestartDeeplink API](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#set-preview-restart-deep-link)
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target/README.md]: - Please remove "API" from the link...should just be `setPreviewRestartDeeplink` that is linked. 
- Please add a period at the end of the sentence.

File: using-mobile-extensions/adobe-target/README.md [hunk 5]
```diff
@@ -357,136 +252,116 @@ purchasedProductIds.add("125");
...
 
-#### **Objective-C Example**
+The `collectLaunchInfo` API is used to enter the visual preview mode. After the visual preview mode is enabled, a red floating button is displayed on the app screen. This button can be pressed to enter the visual preview mode.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target/README.md]: Please merge the sentences as follows:
"To enter the preview visual mode, use the `collectLaunchInfo` API to enable the mode, and click the red floating button that appears on the app screen.

File: using-mobile-extensions/adobe-target/README.md [hunk 6]
```diff
@@ -357,136 +252,116 @@ purchasedProductIds.add("125");
...
 
-```text
-[ACPTarget setPreviewRestartDeepLink:@"myApp://HomePage"];
-```
+{% tabs %}
+{% tab title="Android" %}
 
-#### **Swift Example**
+On Android, when the application is launched as a result of a deeplink, `collectLaunchInfo` API will be invoked internally with the target Activity and deeplink information will be extracted from the Intent extras.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target/README.md]: - Please reword as follows:
"In Android, when the application is launched as a result of a deep link, the `collectLaunchInfo` API is internally invoked, and the target Activity and deep link information is extracted from the Intent extras."

Some questions:
1. Is "target" referring to the solution? Should it be "Target"?
2. Do "Activity" and "Intent" need to be capitalized?

File: using-mobile-extensions/adobe-target/README.md [hunk 7]
```diff
@@ -506,9 +381,14 @@ If you need to update SDK configuration, programmatically, use the following inf
...
+| target.sessionTimeout | Time, in seconds, for which the Target session ID and Egde Host are persisted. If not specified, then default timeout value is 30 minutes. |
+
+{% hint style="warning" %}
+We recommend that you use Experience Platform Launch configuration to pass the property token instead of passing it in as a mbox parameter. If the property token is passed in Experience Platform Launch configuration and also as a mbox parameter, the token that was entered in the mbox parameter is discarded.
+{% endhint %}
 
 {% hint style="warning" %}
-We recommend that you use Experience Platform Launch config to pass the property token instead of passing it in as a mbox parameter. If the property token is passed in Experience Platform Launch configuration and also as a mbox parameter, the token that was entered in the mbox parameter is discarded.
+Currently `target.sessiontimeout` value can only be configured programmatically. For more information, refer [updateConfiguration API](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/configuration/configuration-api-reference#programmatic-updates-to-configuration).
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target/README.md]: Please reword as follows:
"Currently, the `target.sessiontimeout` value can only be configured programmatically. For more information, see [updateConfiguration](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/configuration/configuration-api-reference#programmatic-updates-to-configuration)."

File: using-mobile-extensions/adobe-target/README.md [hunk 8]
```diff
@@ -84,271 +98,152 @@ public class TargetApp extends Application {
...
 {% endtab %}
 {% endtabs %}
 
-### Using Locations Displayed API for Prefetch
 
-In some situations, you might want to want to inform Target that the corresponding location \(mbox\) has been viewed. This API sends a display notification to Target for a prefetched mbox, which helps Target record location display events.
+### Target Parameters
 
-Tip: If you are only using regular mboxes, and not prefetching any mbox content, do not call this method.
+`TargetParameters` encapsulates `mboxParameters`, `profileParameters`, `orderParameters` and `productParameters` and provides an easy way to pass these parameters in a Target request.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target/README.md]: Please reword as follows:
"`TargetParameters` encapsulates `mboxParameters`, `profileParameters`, `orderParameters` and `productParameters` and allows you easily pass these parameters in a Target request. "

File: using-mobile-extensions/adobe-target/README.md (hunk 3 above)
Comment (rymorale) [reply using-mobile-extensions/adobe-target/README.md]: I've added iOS and android specific links on URL scheme setup.

File: using-mobile-extensions/adobe-target/README.md (hunk 6 above)
Comment (rymorale) [reply using-mobile-extensions/adobe-target/README.md]: I've capitalized `Target` and left Activity and Intent capitalized as they are java classes.

**Pull request merged**